### PR TITLE
Avoid getting all types with an empty type name #6039

### PR DESCRIPTION
**Pull request by junichi11**
- https://github.com/apache/netbeans/issues/6039
- Remove `<>` and `{}` from type name
- Ignore an empty type name to avoid getting all types
- Add unit tests

### CODE REVIEW

File: php/php.editor/src/org/netbeans/modules/php/editor/actions/ImportDataCreator.java
Comment (junichi11) [inline php/php.editor/src/org/netbeans/modules/php/editor/actions/ImportDataCreator.java:97]: Before: got all types with an empty type name here. So, the following method was called many times.

https://github.com/apache/netbeans/blob/9e68d3d9630f9403fc14c3a389c803643d92521f/php/php.editor/src/org/netbeans/modules/php/editor/actions/FixDuplicateImportStmts.java#L248-L259

**Review by tmysik — APPROVED** [review]

**Comment by junichi11** [comment]
@tmysik Thank you for your review! I'll wait for being merged by the release team :)

**Pull request merged**
